#116030893 Implement social share buttons

oluosiname opened the pull request:
#### What does this PR do?

It allows customers to share products via social media
#### Description of Task to be completed?

Design social media buttons
Implement buttons to share on social media
#### What are the relevant pivotal tracker stories?
#116030893

Commit:
feat(share product): implement twitter button

design and implement twitter button to share product on twitter

Commit:
feat(share product): design and implement facebook button

design and implement facebook button to share product on facebook

[finishes #116030893]

0duaht commented on app/assets/javascripts/products.js:
```diff
@@ -66,4 +68,29 @@ $(document).ready(function(){
+      fjs.parentNode.insertBefore(js,fjs);
+    }
+  }
+
```
Nice! Could you make the setTweet function more descriptive? Especially with regards to variable names.

Commit:
feat(share product): design and implement google button

design and implement google button to share product on google plus

[finishes #116030893]

makinwab commented:
Feature to be redone.
